sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -269,9 +269,18 @@ class CookieHeaders(object):
 class Versions(object):
```

Comment:
Is this a public class? If so we should not make breaking changes just to make the linter happy.